apps/todos-e2e/src/integration/todo-list.spec.ts

```diff
@@ -1,12 +1,90 @@
-import { getGreeting } from '../support/app.po';
-
 describe('todo list', () => {
   it('should show todo items', () => {
     cy.intercept('/api?query=all', {
-      body: [{ text: 'test TODO 1' }, { text: 'test TODO 2' }],
+      body: [
+        {
+          text: 'test TODO 1',
+          isDone: false,
+          id: 1,
+        },
+        {
+          text: 'test TODO 2',
+          isDone: true,
+          id: 2,
+        },
+      ],
+    });
+    cy.visit('/');
+
+    cy.findAllByRole('checkbox').should('have.length', 2);
+    cy.findByRole('checkbox', { name: 'test TODO 1' })
+      .should('exist')
```

Comment:
Hier finde ich should("be.visible") immer ne nette Feinheit, um sicherzustellen, dass Control auch für den Nutzer im ViewPort sichtbar ist.

Reply:
Geht leider nicht, weil das input opacity: 0 hat.
Da hat uns cypress wohl gleich mal ein a11y-Problem aufgezeigt :)